src/client/app/effects/snippet.ts

```diff
@@ -34,6 +34,7 @@ export class SnippetEffects {
                 .map((snippet: ISnippet) => ({ snippet, mode }))
                 .filter(({ snippet }) => !(snippet == null))
                 .mergeMap(({ snippet, mode }) => this._massageSnippet(snippet, mode, isViewMode))
+                .mergeMap(actions => (actions))
```

Comment:
Discussed offline; this didn't work so sticking with the current solution for now.